examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,680 @@
+    "The specification below will create the following pipeline:\n",
+    "```\n",
+    "             +----> Doctors -----------> SaveDoctors\n",
+    "InputData ---+\n",
+    "             +----> OtherProfessions --> SaveOtherProfessions\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "pipeline = '''\n",
+    "pipeline:\n",
+    "  type: composite\n",
+    "  transforms:\n",
+    "    - type: ReadFromCsv\n",
+    "      name: InputData\n",
+    "      config:\n",
+    "        path: data/people.csv\n",
+    "    - type: Filter\n",
+    "      name: Doctors\n",
+    "      input: InputData\n",
+    "      config:\n",
+    "        language: python\n",
+    "        keep: \"profession == 'doctor'\"\n",
+    "    - type: Filter\n",
+    "      name: OtherProfessions\n",
+    "      input: InputData\n",
+    "      config:\n",
+    "        language: python\n",
+    "        keep: \"profession != 'doctor'\"\n",
+    "    - type: WriteToCsv\n",
+    "      name: SaveDoctors\n",
+    "      input: Doctors\n",
+    "      config:\n",
+    "        path: results/doctors\n",
+    "    - type: WriteToCsv\n",
+    "      name: SaveOtherProfessions\n",
+    "      input: OtherProfessions\n",
+    "      config:\n",
+    "        path: results/other-professions\n",
+    "'''\n",
+    "save_to_file(pipeline, 'pipelines/pipeline-nonlinear-01.yaml')\n",
+    "! python -m apache_beam.yaml.main --pipeline_spec_file=pipelines/pipeline-nonlinear-01.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output are 2 files: `results/doctors-00000-of-00001` and `results/other-professions-00000-of-00001`. Let's see their contents:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! head results/doctors-00000-of-00001"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! head results/other-professions-00000-of-00001"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "Congratulations! You've just run Apache Beam pipelines using YAML.\n",
+    "\n",
+    "For all the available operations visit the documentation: # todo(yaml) add url\n",
+    "\n",
+    "For a list of available transforms, visit # todo(yaml) add url\n",
+    "\n",
+    "To run your pipeline in Dataflow, you'll need to set up your Google Cloud and run the pipeline with the `DataflowRunner`. For more information, follow https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline#run-on-dataflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [],
+   "metadata": {
+    "collapsed": false
+   }
+  }
```

Comment:
nit: Do we need this last blank code block?

Reply:
removed